occjava/src/Geom.i

```diff
@@ -293,3 +305,64 @@ class Handle_Geom_TrimmedCurve : public Handle_Geom_BoundedCurve {
 class Handle_Geom_Plane: public Handle_Geom_Surface {
        Handle_Geom_Plane()=0;
 };
+
+%extend Handle_Geom_Plane {
+	gp_Pnt Location() 
```

Comment:
`Location` must be wrapped in `ElementarySurface` (as in OCC) and renamed to `location`.